src/jack.py

```diff
@@ -1445,6 +1445,34 @@ def shortname(self, shortname):
         _check(_lib.jack_port_set_name(self._ptr, shortname.encode()),
                'Error setting port name')
 
+    @property
+    def aliases(self):
+        """A list of 2 strings containing the aliases for the JACK port."""
+        ctype="char[%d]" % _lib.jack_port_name_size()
+        aliases=[_ffi.new(ctype), _ffi.new(ctype)]
+        aliasesptr=_ffi.new("char *[]", aliases)
+        result=[]
+        if (_lib.jack_port_get_aliases(self._ptr, aliasesptr) > 0):
```

Comment:
The outer parentheses are not necessary.

The "> 0" isn't strictly necessary either, but you can keep it if you think it makes the situation clearer.